sklearn/model_selection/_search.py

```diff
@@ -433,9 +434,8 @@ def _estimator_type(self):
 
     def _more_tags(self):
         # allows cross-validation to see 'precomputed' metrics
-        estimator_tags = self.estimator._get_tags()
         return {
-            'pairwise': estimator_tags.get('pairwise', False),
+            'pairwise': _safe_tags(self.estimator, "pairwise"),
```

Comment:
Why do we need this line? If this is really needed we should add a comment to explain why.

Reply:
Sorry, I misread `_safe_tags(self, "pairwise")` instead of _safe_tags(self.estimator, "pairwise"). Please ignore...